sdk/core/src/main/java/com/klaviyo/core/lifecycle/KlaviyoLifecycleMonitor.kt

```diff
@@ -14,10 +14,6 @@ internal object KlaviyoLifecycleMonitor : LifecycleMonitor, Application.Activity
 
     private var activityObservers = mutableListOf<ActivityObserver>()
 
-    init {
-        onActivityEvent { Registry.log.verbose(it.type) }
```

Comment:
This one wasn't causing any exceptions, but I thought it best to get rid of any unnecessary object inits to head off the same issues.

Reply:
Yeah inits can be notoriously hard to track - better safe than sorry